ui/k-info-panel/switch_info.kytos

```diff
@@ -1,7 +1,9 @@
 <template>
     <k-accordion>
+      <k-button title="Enable" :on_click="bt_enable"></k-button>
```

Comment:
@Alopalao, regarding the layout and show the buttons are being shown, @rmotitsuki had a great suggestion to improve the UX here, to either add `v-if` conditionals in the buttons, so if `switch.enable` is false then it can only be enabled, and similarly if it's true then it can only be disabled. Another alternative would be a toggle button but that might be far slightly harder since the ui doesn't provide that component. 

@rmotitsuki let us know if the conditionals would be ok in this iteration, or if you think it's worth trying to go for this toggle button. thanks

![2022-03-02-100330_410x132_scrot](https://user-images.githubusercontent.com/1010796/156574003-9c171527-3a10-42f7-873f-87e4a40b361f.png)